chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -96,8 +104,38 @@
           <device>/dev/<%= @boot_device %></device>
         <% end %>
         <initialize config:type="boolean">true</initialize>
-        <partitions config:type="list"/>
         <type config:type="symbol">CT_DISK</type>
+        <disklabel>gpt</disklabel>
```

Comment:
do we have to enforce the disklabel to GPT? I'm afraid that this might break non-GPT capable setups (not sure if we still care in 2015, but since this is a maintenance branch..)